include/random_walks/uniform_accelerated_billiard_walk.hpp

```diff
@@ -173,15 +248,15 @@ struct AcceleratedBilliardWalk
             T -= _lambda_prev;
             P.compute_reflection(_v, _p, _update_parameters);
 
-            while (it <= 100*n)
+            while (it <= 1000*n)
```

Comment:
what is the reason behind this change? stability?

Reply:
Better performance in the initialization of the parameters when we have very skinny polytopes.

Now I use `unsigned int _rho = 1000*n;`